src/main/java/core/basesyntax/ArrayList.java

```diff
@@ -1,48 +1,102 @@
 package core.basesyntax;
 
+import java.util.NoSuchElementException;
+
 public class ArrayList<T> implements List<T> {
+    private static final int DEFAULT_CAPACITY = 10;
+    private Object[] dataBase;
+    private int size;
+
+    public ArrayList() {
+        this.dataBase = new Object[DEFAULT_CAPACITY];
+    }
+
+    private void grow(int minCapacity) {
```

Comment:
```suggestion
    private void grow() {
```
you are not using minCapacity.